galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/FrameworkInitialisation.java

```diff
@@ -144,18 +147,24 @@ public FrameworkInitialisation(
         // *** If this is a test run, add the overrides from the run dss properties to
         // these overrides
         if (testrun) {
-            String prefix = "run." + framework.getTestRunName() + ".override.";
-            int len = prefix.length();
+            loadOverridePropertiesFromDss(overrideProperties);
+        }
+    }
 
-            Map<String, String> runOverrides = this.dssFramework.getPrefix(prefix);
-            for (Entry<String, String> override : runOverrides.entrySet()) {
-                String key = override.getKey().substring(len);
+    private void loadOverridePropertiesFromDss(Properties overrideProperties) throws DynamicStatusStoreException {
```

Comment:
Is this duplicated logic with the above ? Why does FrameworkInitialisation have the same logic in as the test runner ?